### ISSUE TITLE
🚀 [Feature Request] - Align the login card texts in center

### ISSUE THREAD

**Issue by MisterMickey** [anmode/grabtern-frontend#348]
### Is your feature request related to a problem ? Please describe. ✍️

The login card is not properly center aligned, I want to fix it.

### Describe briefly how you would implement this enhancement.

```bash
I'll make the login div center align which will eventually make everything aligned perfectly.
```

### Additional context or screenshots

![Screenshot from 2023-06-17 00-54-31](https://github.com/anmode/grabtern-frontend/assets/93720428/54c7879b-fc95-432e-a785-c32d6a441701)

### Are you contributing for a program?

GSSoC2023

### Code of Conduct

- [X] I have read and agree to the Contributing Guidelines of this project.

**Comment by anmode** [comment]
We have updated the login page.. please check develop branch not production.

**Comment by MisterMickey** [comment]
@anmode Where can I see the develop branch preview?

**Comment by anmode** [comment]
> @anmode Where can I see the develop branch preview?
> 

You have to checkout locally...by setting up project

**Comment by anmode** [comment]
> @anmode Where can I see the develop branch preview?
> 

You have to checkout locally...by setting up project

**Comment by anmode** [comment]
We yesterday released our 2.0 version now new changes are live at production 
www.grabtern.com

**Comment by MisterMickey** [comment]
@anmode The login card is still not in center, I wanna fix it, kindly assign me this issue under gssoc23

![Screenshot from 2023-06-18 07-59-06](https://github.com/anmode/grabtern-frontend/assets/93720428/85a85b1b-d8bd-47d3-9ab4-e541a5b90cd2)
